docs/administrator.md

```diff
@@ -127,14 +109,20 @@ kubectl create -f manifests/postgres-operator.yaml
 kubectl create -f manifests/minimal-postgres-manifest.yaml
 ```
 
-### Service account and cluster roles
+### Namespaced service account and roles
+
+For each namespace the operator watches it creates (or reads) a service account
+to be used by the Postgres/Spilo Pods when a new Postgres cluster is deployed.
+This service account is bound to a namespaced Role via RoleBinding, which are
+also created (or read) by the operator. The name and definitions of these
+resources can be [configured](reference/operator_parameters.md#kubernetes-resources).
+Note, that the operator performs **no** further syncing of them.
 
-Note that the service account is named `zalando-postgres-operator`. You may have
-to change the `service_account_name` in the operator ConfigMap and
-`serviceAccountName` in the `postgres-operator` deployment appropriately. This
-is done intentionally to avoid breaking those setups that already work with the
-default `operator` account. In the future the operator should ideally be run
-under the `zalando-postgres-operator` service account.
+Previous releases created RoleBindings to the operator ClusterRole. This can
```

Comment:
It sounds a bit vague, since we're saying "previous releases created RoleBinding to the operator ClusterRole", but do not mention current state of things, just "This service account is bound to a namespaced Role via RoleBinding". Which Role and what is the difference is not clear. Do I read it right, or I'm missing something?

Reply:
I've updated this paragraph saying that until now the clusterrole is referenced by default. Who knows, maybe some are relying on this.